script.js

```diff
@@ -1,4 +1,208 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var nums = [
+  "ace",
+  "two",
+  "three",
+  "four",
+  "five",
+  "six",
+  "seven",
+  "eight",
+  "nine",
+  "ten",
+  "jack",
+  "queen",
+  "king",
+];
+var suitsholder = ["Diamonds", "Hearts", "Clubs", "Spades"];
+var imgholder = [];
+for (let i = 0; i < 13; i += 1) {
+  imgholder.push([]);
+  for (let j = 0; j < 4; j += 1) {
+    imgholder[i].push(`images/${nums[i]}Of${suitsholder[j]}.jpg`);
+  }
+}
+// creating a deck of cards
+const makeDeck = () => {
+  var newDeck = [];
+  for (let i = 1; i <= 13; i += 1) {
+    var suits = ["♦", "♥", "♣", "♠"];
+    for (let j = 0; j < suits.length; j += 1) {
+      var name = `${i}`;
+      if (name === "11") {
+        name = "Jack";
+      } else if (name === "12") {
+        name = "Queen";
+      } else if (name === "13") {
+        name = "King";
+      }
+      var value = parseInt(name);
+      if (name === "Jack" || name === "Queen" || name === "King") {
+        value = 10;
+      }
+
+      var card = {
+        value: value,
+        suit: suits[j],
+        img: imgholder[i - 1][j],
+        name,
+      };
+      newDeck.push(card);
+    }
+  }
+  return newDeck;
 };
+// Get a random index ranging from 0 (inclusive) to max (exclusive).
+const getRandomIndex = (max) => Math.floor(Math.random() * max);
+// Shuffle an array of cards
+const shuffleCards = (deck) => {
+  // Loop over the card deck array once
+  for (let currentIndex = 0; currentIndex < deck.length; currentIndex += 1) {
+    // Select a random index in the deck
+    const randomIndex = getRandomIndex(deck.length);
+    // Select the card that corresponds to randomIndex
+    const randomCard = deck[randomIndex];
+    // Select the card that corresponds to currentIndex
+    const currentCard = deck[currentIndex];
+    // Swap positions of randomCard and currentCard in the deck
+    deck[currentIndex] = randomCard;
+    deck[randomIndex] = currentCard;
+  }
+  // Return the shuffled deck
+  return deck;
+};
+
+//define global variables
+var playerCardPlaceholders = document.getElementsByClassName("p-card");
+var computerCardPlaceholders = document.getElementsByClassName("c-card");
+var deck = makeDeck();
+var shuffledDeck = shuffleCards(deck);
+var playButton = document.getElementById("play-button");
+var hitButton = document.getElementById("hit-button");
+var standButton = document.getElementById("stand-button");
+var restartButton = document.getElementById("restart-button");
+var outputMessageElement = document.getElementById("output");
+
+var playerCards = [];
+var computerCards = [];
+var playerScore = 0;
+var computerScore = 0;
+
+//defind helper functions
+function checkBlackJacket(num1, num2) {
```

Comment:
It will be more ideal for this function to return a boolean instead of updating the score variable.